.github/workflows/stale_pr.yml

```
@@ -28,7 +28,7 @@ jobs:
           days-before-pr-close: 0
           # only run the action on merge conflict PR
           any-of-labels: 'Merge Conflict'
-          exempt-pr-labels: 'internal,Never stale,ON HOLD,WIP'
+          exempt-pr-labels: 'Internal,Never Stale,On Hold,WIP'
```

I could be a stickler here and suggest a space after each comma, but.. Only if you really want to - and maybe we always set it without the space because stalebot doesn't like it otherwise. 🤷‍♂️

```suggestion
          exempt-pr-labels: 'Internal, Never Stale, On Hold, WIP'
```

---

Yeah, forget it - their default doesn't have spaces in it. Best to err on the side of caution

https://github.com/actions/stale#exempt-pr-labels

---

Haha, I think it's working with spaces as well. This is how we have it in `core` and I think it's working there.
https://github.com/pi-hole/pi-hole/blob/ac2f63b138bfd9f73ceb6746e84ab0c46fbc69a4/.github/workflows/stale.yml#L23